workflows/InsightIDR_Enrich_Alert_Data_with_Open_Source_Plugins/workflow.spec.yaml

```
@@ -0,0 +1,22 @@
+extension: workflow
+product: ["insightconnect"]
+name: InsightIDR_Enrich_Alert_Data_with_Open_Source_Plugins
+title: "InsightIDR Enrich Alert Data with Open Source Plugins"
+description: "Workflow that attempts to enrich all following IoCs (IPs, Domains, URLs, Hashes), if available. Can be associated with almost any alert."
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+tags:
+- insightidr
+- enrichment
+- analysis
+- hash
+- domain
+hub_tags:
+  use_cases: []
```

Needs use cases from the use case list.

```suggestion
  use_cases: [soc_automation, siem, threat_detection_and_response, threat_intel]
```